app/addons/documents/resources.js

```diff
@@ -506,7 +506,7 @@ Documents.IndexCollection = PagingCollection.extend({
         view = app.utils.safeURLName(this.view),
         url = FauxtonAPI.urls('view', context, database, design, view);
 
-    return url + query;
+    return (url.indexOf("?") > -1) ? url + "&" + query : url + "?" + query;
```

Comment:
Can you change this to use es6 template strings https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals